matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/crypto/CryptoService.kt

```diff
@@ -162,4 +163,9 @@ interface CryptoService {
      * send, in order to speed up sending of the message.
      */
     fun prepareToEncrypt(roomId: String, callback: MatrixCallback<Unit>)
+
+    /**
+     * Share all inbound sessions of the last chunk messages to the provided userId devices
+     */
+    fun sendSharedHistoryKeysToLastChunk(roomId: String, userId: String, sessionInfoSet: Set<SessionInfoPair>?)
```

Comment:
Can we use same name as js-sdk? would be easier when investigating/checking code.
Also it's probably possible to invite several people at once, so it could be a list of userId?

Reply:
It will be triggered for every userId, so there is no need for a list here